lib/server/imgix.js

```diff
@@ -18,7 +18,9 @@ export const imgixResolver = async (parent, { input }, context) => {
   const parentFSDocumentId = parent._id;
   if (!parentFSDocumentId) return null;
 
-  const fullParentFSFile = await FSCollection.loader.load(parentFSDocumentId);
+  const fullParentFSFile = FSCollection.findOne({
```

Comment:
I don't have context for this but wondering if it would be better to invalidate the cache when we need to instead of always bypassing it